AxonDeepSeg/integrity_test.py

```diff
@@ -62,3 +66,26 @@ def integrity_test():
         # Else, there is a problem in the installation
         print("Integrity test failed... ")
         return -1
+
+def main():
+    # argparse CLI arguments, default is to run the integrity test only
+
+    parser = argparse.ArgumentParser(description='Run AxonDeepSeg tests.')
+
+    parser.add_argument('--full', action='store_true', help='Run the the full test suite.'
+                        'If not specified, then only the integrity tests will be run.')
+
+    args = parser.parse_args()
+
+    if args.full:
+        # Run the full test suite
+        sys.exit(pytest.main(["-x", Path(__file__).resolve().parent.parent / "test"]))
```

Comment:
Just out of curiosity, why have we specified the `-x` option here?

> ```
>   -x, --exitfirst       Exit instantly on first error or failed test
> ```

If a user runs `axondeepseg_test --full` and there are multiple errors, then it might end up wasting a lot of time when troubleshooting with the users, as the errors would have to be solved iteratively, 1 by 1, since only one error would be shown at a time.

Reply:
Nevermind my last comment, i didnt fully read yours. It came from an example for using pytest within python, i thought it was used for another reason. Ill change it!

Reply:
Done!